backend/app/services/implementations/residents_service.py

```diff
@@ -21,6 +22,27 @@ def __init__(self, logger):
         """
         self.logger = logger
 
+    def to_residents_json_list(self, resident_results):
+        residents_json_list = []
+        for result in resident_results:
```

Comment:
so what you can here is split up the result:
`resident, building = result[0], result[1]`

and then use `to_dict()`

`resident_dict = resident.to_dict()`

and then just add the building:
resident_dict["building"] = building